### PR TITLE
added new file

### PR DESCRIPTION
**Pull request by krishkatyal**
password manager basic project

### CODE REVIEW

File: password_manager.c [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                return 0;
+            default:
+                printf("Invalid choice.");
+                sleep(3);
+                break;
+        }
+    }
+}
+Footer
+© 2022 GitHub, Inc.
```
Comment (mhmdsami) [inline password_manager.c]: What it this? 👀

Comment (krishkatyal) [reply password_manager.c]: that is for when the code ends haha

Comment (krishkatyal) [reply password_manager.c]: i have updated it

**Comment by mhmdsami** [comment]
This does not align with the purpose of this repo, so I am afraid I will not be able to merge this. Thanks anyway!

**Pull request closed without merging**